cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -32,10 +32,23 @@
         inverse="_inverse_is_global",
         store=True,
     )
-
-    value_char = fields.Char(string="Value")
     note = fields.Text(related="variable_id.note", readonly=True)
     active = fields.Boolean(default=True)
+    variable_type = fields.Selection(
+        selection=[("s", "String"), ("o", "Options")],
+        string="Variable Type",
+        related="variable_id.variable_type",
+        store=True,
+        readonly=True,
+    )
+    option_id = fields.Many2one(
+        comodel_name="cx.tower.variable.option",
+        string="Option",
+        domain="[('variable_id', '=', variable_id)]",
+    )
```

Comment:
@nikola-vlavceski-solisa have a look at this. Although this is not possible from UI because of the domain, it may happen if someone will do this with code.